Cargo.toml

```diff
@@ -7,6 +7,9 @@ members = [
     "packages/utils/crates/utils-metrics"
 ]
 
+[workspace.dependencies]
```

Comment:
When specifying it like this, please also add `wasm-opt` since this is required by `wasm-pack` because we set `opt-level = "s"`

Reply:
Where/how is `wasm-opt` installed so far?

Reply:
using `yarn add wasm-pack`